### ISSUE TITLE
Random cropping during preprocessing in train.py

### ISSUE THREAD

**Issue by mgarbade** [isht7/pytorch-deeplab-resnet#20]
Thanks a lot for your repository. While reading through `train.py` I noticed that PASCAL VOC images are being resized to 321x321 in [this line](https://github.com/isht7/pytorch-deeplab-resnet/blob/master/train.py#L107). 
In the original paper they perform a random cropping on img and gt mask. [Here](https://github.com/DrSleep/tensorflow-deeplab-resnet/blob/master/deeplab_resnet/image_reader.py#L41-L70) is an example from the corresponding Tensorflow code. There mask and image are concatenated and then cropped simultaneously.

**Comment by isht7** [comment]
Thank you for your feedback! I will try to add this in some time.
